public/app/common/helpers/poll/index.js

```diff
@@ -0,0 +1,43 @@
+function Poll (options) {
+
+  this.options = options;
+  this.options.predicate = this.options.predicate || function truthSayer () { return true; };
```

Comment:
As `truthSayer` is a noun (and thus a class name), we could call the method here something like `forever` or `alwaysTrue`